src/components/partner/partner.service.ts

```diff
@@ -30,6 +36,8 @@ export class PartnerService {
     @Logger('partner:service') private readonly logger: ILogger,
     @Inject(forwardRef(() => AuthorizationService))
     private readonly authorizationService: AuthorizationService,
+    @Inject(forwardRef(() => ProjectService))
```

Comment:
Only do this if it's necessary. If it'll inject without it, omit it.

Reply:
Yeah, I get a circular dependency without it.  ☹️